lib/src/string_codecs.dart

```diff
@@ -81,7 +81,7 @@ class CamelCaseEncoder extends Converter<List<String>, String> {
   }
 }
 
-final dashesToCamelCase = new SeparatorCodec("-", splitter: new RegExp(r'[_-]'))
+Codec dashesToCamelCase = new SeparatorCodec("-", splitter: new RegExp(r'[_-]'))

```

Comment:
As a note of caution, `final` is a keyword, not a type.  Changing this may cause bugs if other areas depend on`dashesToCamelCase` being immutable.  Are you certain this shouldn't be `final Codec dashesToCamelCase` instead?